src/ng/http.js

```diff
@@ -1249,7 +1249,11 @@ function $HttpProvider() {
           cache,
           cachedResp,
           reqHeaders = config.headers,
-          url = buildUrl(config.url, config.paramSerializer(config.params));
+          // origUrl could be a $sce trusted URL which used a wrapper class
+          origUrl = config.url,
+          url = $sce.valueOf(origUrl);
```

Comment:
There is! I just found it. All the tests in this PR are only testing the `$httpBackend` so none of the changes in this file are tested.

I am working on improving that :-)